### PR TITLE
infinite med

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
About The Pull Request
Makes every basic pill bottles and autoinjectors infinite.
Gives marines a special webbing with 1 slot less and medkit pouch with 2 slot less as well.
Medical jaeger and normal corpsman webbing and medkit pouch, both of which remain with same slot are now Squad Corpsman locked now.

Why It's Good For The Game
Having marines choose between their space with ammo and meds as well as what meds they should consider taking is a much better choice than making them choose the meds that might have ran out. Currently anyone who joins early can take away most meds and late joiners never get meds in spite of having a loadout saved. This fixes those balance issues by decreasing the total space available for meds while making them infinite on vendor.

@David-Stormwell and @BraveMole approved this pr idea so did @TiviPlus I believe.

If I am missing any chance/you have opinion on something being limited/more nerfed comment on the pr.

Changelog
🆑
add: Added marine corpsman webbing with 5 slots[originally 6 slots].
add: Added marine medkit pouch with 5 slots[originally 7 slots].
del: Medical jaeger exosuit storage module as well as normal webbing and medkits are now Squad Corpsman locked.
balance: Most medical pill bottles and autoinjectors are now infinite.
/🆑

### CODE REVIEW

**Comment by AnCopper** [comment]
Lol what.

**Comment by Captiajoe** [comment]
so what you are telling me is that in exhange for some of the better chems and extra peri+ and qc+ pouches i am gonna get the medkit pouches that have 7 slots (which i usually get from spare vendor) and gives me only splints and trauma kits and burn kits, is this what you are telling me? (???????)

**Comment by joniroq** [comment]
The Medkit Pouch and the _Medkit Pouch_ are identically named? which is only gonna confused people unless they examine

**Comment by SpaceLoveSs13** [comment]
> so what you are telling me is that in exhange for some of the better chems and extra peri+ and qc+ pouches i am gonna get the medkit pouches that have 7 slots (which i usually get from spare vendor) and gives me only splints and trauma kits and burn kits, is this what you are telling me? (???????)

No with this PR you wont be getting 7 slot medkit pouch anymore. Only 5 slots. The 7 slot will be locked for medics only. Same for corpsman webbing.

**Comment by jroinc** [comment]
I guess I'm gonna have to nerf marine meds into the ground now under the assumption that every unga is on them all the time.

**Comment by spookydonut** [comment]
@jroinc that comment is not acceptable, consider this a warning.

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -532,6 +538,8 @@
 	new /obj/item/storage/pill_bottle/tramadol(src)
 	new /obj/item/stack/medical/splint(src)
 
+
+
```
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:541]: useless new lines

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 2]
```diff
@@ -648,6 +648,7 @@
 		/obj/item/reagent_containers/hypospray/advanced/oxycodone,
 		/obj/item/storage/firstaid/adv,
 		/obj/item/clothing/glasses/hud/health,
+
```
Comment (BraveMole) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:651]: useless new line

File: code/game/objects/items/storage/pouch.dm [hunk 3]
```diff
@@ -512,6 +512,11 @@
 		/obj/item/reagent_containers/hypospray,
 	)
 
+/obj/item/storage/pouch/medkit/corpsman
```
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:515]: you're not giving this to anyone

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/pouch.dm (hunk 3 above)
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:515]: It's not full like it should.

File: code/game/objects/items/storage/pouch.dm [hunk 4]
```diff
@@ -512,6 +512,11 @@
...
+	name = "Corpsman medkit pouch"
+	desc = "A standard use medkit pouch that can contain all kinds of stuff.This Marine version has 7 slots."
+	storage_slots = 7
+
 /obj/item/storage/pouch/medkit/full/Initialize()
```
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:520]:
```suggestion
/obj/item/storage/pouch/medkit/corpsman/Initialize()
```

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 2 above)
Comment (BraveMole) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:651]: standing issue

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by QualityVan** [comment]
The primary issue you're describing isn't actually about a shortage of supplies, it's about resource mismanagement. As is, non-corps marines are assured available meds via pouch selection. If those options aren't good enough, buff them. Personally, I'd rec swapping the ointment, gauze, and tram injector in first aid pouches out for bkt pill packets, leaving in the tricord and splints.

File: code/game/objects/items/storage/pouch.dm [hunk 5]
```diff
@@ -493,12 +493,12 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
 
 /obj/item/storage/pouch/medkit
-	name = "medkit pouch"
-	desc = "A standard use medkit pouch that can contain all kinds of stuff."
+	name = "Marine medkit pouch"
+	desc = "A standard use medkit pouch that can contain all kinds of stuff.This Marine version has 5 slots."
```
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:497]:
```suggestion
	desc = "A standard use medkit pouch that can contain all kinds of stuff. This Marine version has 5 slots."
```

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 2 above)
Comment (BraveMole) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:651]: This is STILL a standing bloody issue, remove that line that you added

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Review by BraveMole — APPROVED** [review]

**Comment by AndroidMK2** [comment]
I just say I don't like this change.

**Comment by Losenis** [comment]
If chems are too powerful then just buff the antichem benos

**Comment by jroinc** [comment]
> 
> 
> @jroinc that comment is not acceptable, consider this a warning.

If infinimeds are added, I'll treat ungas as though they're on all meds all the time. This will need some changes.

**Comment by EvilMelonTheSecond** [comment]
> 
> 
> > @jroinc that comment is not acceptable, consider this a warning.
> 
> If infinimeds are added, I'll treat ungas as though they're on all meds all the time. This will need some changes.

I already get BKT with the current limited meds cuz I rush loadouts for em. I can say for sure that 100% uptime is not sustainable on the frontlines even with the current storage and even then xeno will burst me down rapidly when I have meds. This PR has been needed for a long time in my opinion.

**Comment by jroinc** [comment]
> 
> 
> > > @jroinc that comment is not acceptable, consider this a warning.
> > 
> > 
> > If infinimeds are added, I'll treat ungas as though they're on all meds all the time. This will need some changes.
> 
> I already get BKT with the current limited meds cuz I rush loadouts for em. I can say for sure that 100% uptime is not sustainable on the frontlines even with the current storage and even then xeno will burst me down rapidly when I have meds. This PR has been needed for a long time in my opinion.

With my experience, there are many ungas who routinely have 100+ units of chemicals in them while fighting. This is now sustainable long-term because *stuff a backpack with pill bottles* same as people used to do with revolvers. I'mma nerf long-term sustain some, if it becomes an issue.

Which it will.

**Comment by Losenis** [comment]
> With my experience, there are many ungas who routinely have 100+ units of chemicals in them while fighting.

And wth my experience treating the octodecigorillion ungas while running medical, 100 of those units are either neurotox or killer ODs.

**Comment by jroinc** [comment]
Yeah, that doesn't line up with what I see by chemical breakdown, *but* I still am gonna collect more evidence first.

**Comment by bowlofcereal** [comment]
I don't know how this solves anything or makes anything remotely better. Hell, I don't even know what the problem here is in this instance, it's a chore to decipher this PR.

**Comment by Alexeivichi** [comment]
Honestly, as much as I'd love this, it'll throw a certain balance off: The supply and demand for healing chems, and consequences. This is actually a pretty strong buff, and marines will always find a way to maximize their play. This will enable a lot of things that would be "bad" for the meta, IMO.

It'll be far better to increase the amount of basic pills roundstart. Adding lesser version of existing storage items for medical items would result in people relying on medical players for the better versions, or even result in them greytiding as usual moreso. I could be wrong but if I recall correctly: This would also result in "reverse powercreep". Correct me on this, I'm 90% I'm wrong though. Pretty sure people can save the better items in their Loadout Managers and spawn them that way.

On the playing field, marines would have **less of a reason to conserve pills** other than having to return to the F.O.B and resupply. This would be a heavy buff and is certain to piss xenomorph players off. 

Heck it could even result in harder fobbiting and harder sieges of F.O.B. I believe healing chemicals should be in limited supply, and if you need more: Ask Shipside medbay to produce more. (Just like #8267 made advance kits and splints limited, we had to learn to use it better and more sparingly, and spend Requisition points if we needed more).

Anywho, enough passive rambling: This PR would honestly be better if #8358 were to get implemented, but I sadly have to disapprove of this.

**Comment by David-Stormwell** [comment]
Question, why can't instead of having infinite meds, make it easier to order them via req? 
@BraveMole Perhaps a order button on marine meds to request a type of chem? Just a alternative instead of having this + req factorio exists now which might be able to be linked and solve this issue better.

**Pull request closed without merging**